src/ImageSharp/Formats/Jpeg/JpegDecoderCore.cs

```diff
@@ -491,7 +488,8 @@ internal void ParseStream(BufferedReadStream stream, SpectralConverter spectralC
                 }
 
                 // Read on.
-                fileMarker = FindNextFileMarker(this.markerBuffer, stream);
+                fileMarker = FindNextFileMarker(stream);
+                Console.WriteLine($"Found marker: {fileMarker.Marker} at {fileMarker.Position}");
```

Comment:
This Console.WriteLine is probably a leftover from debugging

Reply:
Ups!